tests/installation/installation_overview.pm

```diff
@@ -16,12 +16,24 @@ use warnings;
 use base "y2logsstep";
 use testapi;
 
+sub sle15_workaround_broken_patterns {
+    if (check_var('VERSION', '15')) {    # SLE 15 has pattern errors, workaround them - rbrown 04/07/2017
+        while (check_screen('sle-15-failed-to-select-pattern')) {
+            record_soft_failure 'bsc#1047327';
+            send_key 'alt-o';
+        }
```

Comment:
I believe Oliver suggests to create an array of tags and perform assert screen against it, and then use match_has_tag to see which screen it is. It doesn't stop us from having a while loop there. But then we need to find some exit condition. It's nice to have, as if test fails, you see set of tags we expect. We have similar approach when process warning popup during installation, as some are expected.